enterprise/backend/src/metabase_enterprise/advanced_config/api/logs.clj

```diff
@@ -0,0 +1,42 @@
+(ns metabase-enterprise.advanced-config.api.logs
+  "/api/logs endpoints.
+
+  These endpoints are meant to be used by admins to download logs before entries are auto-removed after the day limit.
+
+  For example, the `query_execution` table will have entries removed after 30 days by default, and admins may wish to
+  keep logs externally for longer than this retention period."
+  (:require
+   [clojure.string :as str]
+   [compojure.core :refer [GET]]
+   [malli.core :as mc]
+   [malli.transform :as mtx]
+   [metabase.api.common :as api]
+   [metabase.db.connection :as mdb.connection]
+   [metabase.util.malli :as mu]
+   [metabase.util.malli.schema :as ms]
+   [toucan2.select :as select]))
```

Comment:
Most common functions are exposed via `toucan2.core` so you can just require it as `t2` and call `t2/select` (that's the convention I've seen elsewhere, at least)